app/app/src/main/java/me/bickositieff/raspio/ui/models/Song.kt

```diff
@@ -13,4 +13,21 @@ class Song(
     fun durationToString(): String{
         return ""+duration/60 + ":" + duration%60
     }
+
+    override fun equals(other: Any?): Boolean {
```

Comment:
In Kotlin you should use data classes instead of generated methods. See https://kotlinlang.org/docs/data-classes.html